src/web/components/Navigation/SideNav.tsx

```diff
@@ -16,14 +17,21 @@ function MenuItem({
   linkClass,
   isHidden,
 }: Pick<PortalRoute, 'path' | 'description' | 'isHidden'> & { linkClass?: string }) {
+  const { participantId } = useParams();
+  const pathWithParticipant = getPathWithParticipant(path, participantId);
```

Comment:
We have to do this because the paths have `:participantId` in them - react-router is not smart enough to swap it out when defining the paths like this.